src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -128,6 +145,9 @@ module sonic-acl {
 					}
 				}
 
+				/* Validating 'SRC_IP' exist if ACL type is 'INNERSRCMACREWRITE' */
```

Comment:
not sure we need this, INNER_SRC_IP is a generic match type, can be applicable to other use cases for acl_rule, we probably don't need to mandate/limit it to innersrcmacrewrite